docs/network.md

```diff
@@ -0,0 +1,44 @@
+# Network 
+
+## user-mode network (192.168.5.0/24)
+
+By default Lima only enables the user-mode networking aka "slirp".
+
+### Guest IP (192.168.5.15)
+
+The guest IP address is typically set to 192.168.5.15.
+
+This IP address is not accessible from the host by design.
+
+Use `vde_vmnet` to allow accessing the guest IP from the host and other guests.
+
+### Host IP (192.168.5.2)
+
+The loopback addresses of the host is accessible from the guest as 192.168.5.2.
+
+### DNS (192.168.5.3)
+
+The DNS.
+
+## `vde_vmnet` (192.168.105.0/24)
```

Comment:
Add a note that the IP range is just the default setting and can be configured on the vde_vmnet side?

Reply:
updated